java/HealthVault Mobile/HVSample/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,237 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+	xmlns:design="http://schemas.android.com/apk/res-auto"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
+	android:background="#e3e3e3"
+    android:orientation="vertical"
+    tools:context=".MainActivity">
+
+    <LinearLayout
```

Comment:
I have created task # 54619 for this comment

---
In reply to: [112366563](https://github.com/Microsoft/healthvault-samples/pull/4#discussion_r112366563) [](ancestors = 112366563)